runtime/src/core/closures.js

```diff
@@ -1,7 +1,5 @@
 import { managedMemory, grainModule } from "../runtime";
-import { throwGrainError } from "../errors/errors";
 import { grainToJSVal, JSToGrainVal } from "../utils/utils";
-import { GRAIN_ERR_ARITY_MISMATCH } from "../errors/error-codes";
```

Comment:
Do we still use `../errors/errors` or `../errors/error-codes` or can those be deleted now?